packages/core-interfaces/ipfs/.gitignore

```diff
@@ -0,0 +1,3 @@
+build
```

Comment:
Hey @pileks, no need to have `.gitignore` or `.nvmrc` files in the packages within the monorepo, since these are typically covered by the root monorepo files.

If there are special needs by the package though, that's when you need to create one at the sub-module level.